### DISCUSSION TITLE
[2023-06-12] Library and Engine Team Update

### DISCUSSION THREAD

**Discussion by JaroslavTulach** [enso-org/enso#7016] [Weekly Demo Day]
Greg:
- #6886
- #6424 

Radek:
- #6990
- #6964

James:
- #6973 
	- QA and various tweaks
	- type detection for values in Mixed columns, allowing column operations expecting a specific type to run when a Mixed type column is provided but all values fit the desired type
	- ability to list AWS profiles and S3 buckets
- date/time parsing improvements (demo next week)

Hubert:
- #6857 
- #6897 
- #6841
- #7005 

Dmitry:
- #6900
- #6903
- #6936 

Jaroslav:
- #6955 
- #6821 
- #5500
- #7009

Pavel
- #6431
- https://github.com/enso-org/enso/pull/6931

**Comment by Akirathan** [comment]
![image](https://github.com/enso-org/enso/assets/14013887/f3b9eb9f-f473-4b35-a475-652df3f249e8)

**Comment by Akirathan** [comment]
![image](https://github.com/enso-org/enso/assets/14013887/e5443016-ea9b-4f85-9ec5-b074c2dcca8a)
